web/src/components/profileEdit/QuotaCardEdit.tsx

```diff
@@ -36,7 +41,18 @@ interface IQuotaCardEditProps {
   hasPendingEdit: boolean;
 }
 
+const StyledQuotaEditContainer = styled.div`
+  max-width: 35vw;
+  padding: 0px;
+  @media only screen and (max-width: 680px) {
+    max-width: 90vw;
+  }
+`;
+
 const QuotaCardEdit: React.FC<IQuotaCardEditProps> = (props) => {
+  // @ts-ignore
+  const required = (value) => (value ? undefined : 'Required');
```

Comment:
I recognize this was likely copied from the CreateFormRequest card, but that `// @ts-ignore` can be removed and the circumstance resolved with the following change.
```suggestion
  const required = (value: boolean) => (value ? undefined : 'Required');
```